client/src/ui/organisms/placement/index.tsx

```diff
@@ -114,24 +128,65 @@ export function TopPlacement() {
           extraClassNames={["top", "container"]}
           cta={placementData.top?.cta}
           imageHeight={50}
+          renderer={RenderSideOrTopBanner}
         ></PlacementInner>
       )}
     </div>
   );
 }
 
+export function HpMainPlacement() {
+  return HpPlacement({ imageWidth: 970, imageHeight: 250 });
+}
+
+export function HpFooterPlacement() {
+  return HpPlacement({ imageWidth: 728, imageHeight: 90 });
+}
+
+function HpPlacement({
+  imageWidth,
+  imageHeight,
+}: {
+  imageWidth: number;
+  imageHeight: number;
+}) {
+  const placementData = usePlacement();
+  const { backgroundColor } = placementData?.hpMain?.colors || {};
```

Comment:
Does this, and other references to `hpMain` in this function need to be like `[location]` where `location` is an argument of type `"hpMain" | "hpFooter"`?

Otherwise, we seem to be trying to render the hpMain placement in the footer too.